Automattic/jetpack
# Admin Page Fix all eslint warnings about the react/jsx-no-bind rule

Fixes all eslint warnings about the react/jsx-no-bind rule.

#### Changes proposed in this Pull Request:

* Upgrades the `react/jsx-no-bind rule` to an error.
* Refactors several components to drop `.bind()` and `onEvent={ () => {} }` patterns among jsx code.

#### Testing instructions:

* Run `npm run lint` and expect to see no linting warnings.
* Build the admin page and confirm it does it properly (`yarn build`).
